addons/mail/tests/common.py

```diff
@@ -1507,3 +1507,11 @@ def _generate_attachments_data(count, res_model, res_id, attach_values=None, pre
             'res_id': res_id,
             **attach_values,
         } for x in range(count)]
+
+    def _filter_persona_fields(self, data):
```

Comment:
Would say to move in ``TestChannelInternals`` class, as it has globally no use on standard mail common class (it is only a trick for some specific discuss tests). Purpose is to try to keep it as small as possible (well it is already not that small :D ), and keep data / tools only where it is necessary (can be moved afterwards if necessary anyway). Cheers !

Reply:
I see the point, I have a following PR coming soon where I'll need this in livechat as well, which uses the same common case so that felt more natural to have it there.